### DISCUSSION TITLE
Install path on Linux

### DISCUSSION THREAD

**Discussion by ami-GS** [microsoft/msquic#2163] [Q&A] [answered]
Hi Team,

In case of Linux, Why msquic library and headers are installed under /usr/local/msquic (I want to emphasize hardcoded "/msquic" in CMakeLists.txt)
In this case developers need to set environment variable or edit Makefile to make compiler/linker to find these.
It would be happy if we can specify full path without "/msquic", like /usr/lib for library, /usr/include for headers.

I found there are a lot of comments in CMakeLists.txt around the part where specifying installation path. So I thought you guys have some thought.

**Comment by thhous-msft** [comment, marked as answer]
We'd happily accept a PR to make that configurable. When using cmake's targets for linking, having them in a subfolder works well. But I agree that its a bit more difficult if you just want to link straight from the compiler without cmake.

**Reply by ami-GS** [reply]
I will create PR, thanks!
